controllers/dbaas.redhat.com/configmap.go

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (C) 2020 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package dbaasredhatcom
+
+import (
+	"context"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+)
+
+const (
+	PROVIDERDATAVALUE = "Crunchy Bridge managed PostgreSQL"
+	PROVIDERDATAKEY   = "provider"
+
+	INVENTORYDATAVALUE    = "CrunchyBridgeInventory"
+	INVENTORYKINDDATAKEY  = "inventory_kind"
+	CONNECTIONKINDDATAKEY = "connection_kind"
+	CONNECTIONDATAVALUE   = "CrunchyBridgeConnection"
+	CREDENTIALS_FIELDS    = "credentials_fields"
```

Comment:
This one var here using an underscore for spacing when the rest just put the words together bothers me :) Should we make this `CREDENTIALFIELDS` or maybe add underscore spacing to the others instead?